src/scenes/details/Details.js

```diff
@@ -14,14 +14,33 @@ class Details extends Component {
     headerTitleStyle: homeStyles.text
   };
 
+  markAsDone = (id) => {
+    const { navigation } = this.props;
+    navigation.navigate('Home', {
+      idTask: id,
+    });
+  }
+
   render() {
     const { navigation } = this.props;
-    const task = navigation.getParam("task");
+    const task = navigation.getParam('task');
     return (
       <View style={homeStyles.listContainer}>
         <View style={detailStyles.container}>
+          <Text style={detailStyles.notDone}>
+            {task.done ? 'Done' : 'Not Done'}
+          </Text>
           <Text style={detailStyles.title}>{task.title}</Text>
           <Text style={detailStyles.description}>{task.description}</Text>
+          {!task.done && 
+            <TouchableOpacity onPress={() => this.markAsDone(task.id)}>
```

Comment:
Please avoid using functions inside jsx. The reason is because always you render the page the function is going to be created again and that its bad to the performance